### PR TITLE
Changes to include supported TRT IO datatypes in testing

### PR DESCRIPTION
**Pull request by tanmayv25**
Enables the testing framework to include coverage for other supported IO data types by TRT. With the requirement of 3d tensors relaxed for datatypes except INT8, the special handling of TRT in our testing framework is somewhat relaxed as well.

### CODE REVIEW

File: qa/L0_batcher/batcher_test.py [hunk 1]
```diff
@@ -69,7 +69,7 @@ def check_response(self, trial, bs, thresholds,
                                use_grpc=False, skip_request_id_check=True,
                                use_streaming=False)
             elif trial == "plan":
-                tensor_shape = (input_size,1,1)
+                tensor_shape = (input_size)
```
Comment (deadeyegoodwin) [inline qa/L0_batcher/batcher_test.py]: (input_size,)

File: qa/L0_lifecycle/lifecycle_test.py [hunk 2]
```diff
@@ -193,7 +193,7 @@ def test_parse_error_model_no_version(self):
                 iu.infer_exact(self, model_name, tensor_shape, 1,
                                np.float32, np.float32, np.float32, swap=True)
             for version in [1, 3]:
-                iu.infer_exact(self, 'plan', (input_size, 1, 1), 1,
+                iu.infer_exact(self, 'plan', (input_size), 1,
```
Comment (deadeyegoodwin) [inline qa/L0_lifecycle/lifecycle_test.py]: (input_size,)

File: qa/L0_batcher/batcher_test.py (hunk 1 above)
Comment (tanmayv25) [reply qa/L0_batcher/batcher_test.py]: fixed

File: qa/L0_lifecycle/lifecycle_test.py (hunk 2 above)
Comment (tanmayv25) [reply qa/L0_lifecycle/lifecycle_test.py]: fixed

**Pull request merged**